scripts/common/rook.sh

```diff
@@ -67,46 +67,63 @@ function remove_rook_ceph() {
     # make sure there aren't any PVs using rook before deleting it
     all_pv_drivers="$(kubectl get pv -o=jsonpath='{.items[*].spec.csi.driver}')"
     if echo "$all_pv_drivers" | grep "rook" &>/dev/null ; then
-        # do stuff
-        printf "%b" "$RED"
-        printf "ERROR: \n"
-        printf "There are still PVs using rook-ceph.\n"
-        printf "Remove these PVs before continuing.\n"
-        printf "%b" "$NC"
-        exit 1
+        logFail "There are still PVs using rook-ceph."
+        logFail "Remove these PVs before continuing."
+        return 1
```

Comment:
@emosbaugh @laverya @ricardomaraschini @rrpolanco 

 I do not think that we need to stop the upgrade because was not possible to remove Rook. IMO we just make clear that printing in red so that users are aware of and we have a chance to do the upgrade with success and afterwords remove rook manually.

Any reason for we need to bail the script in this scenario?